spring-cloud-vault-core/src/main/java/org/springframework/cloud/vault/VaultResponse.java

```diff
@@ -16,11 +16,11 @@
 
 package org.springframework.cloud.vault;
 
-import java.util.Map;
+import lombok.Data;
```

Comment:
Here's the link to [eclipse-code-formatter.xml](https://github.com/spring-cloud/spring-cloud-build/blob/master/spring-cloud-dependencies-parent/eclipse-code-formatter.xml).